### PR TITLE
[2.9] Symlink docker machine driver binaries

### PR DESCRIPTION
**Pull request by pjbgf**
The docker machine drivers for `linode` and `harvester` are located in two directories: `/opt/drivers/management-state/bin` and `/usr/share/rancher/ui/assets/`. The changes replace the latter with a symlink to the former.

The impact of duplicating those binaries is twofold:
- Increased container image size.
- Double counting of CVE instances by CVE scanners.

After the changes it was confirmed that the files are still accessible via `<rancher-url>/assets/<driver-name>` and that CVE scanners are no longer double counting CVEs.

### CODE REVIEW

**Review by macedogm — APPROVED** [review]

**Comment by macedogm** [comment]
Note: I pinged internally some folks from Hostbusters to get a second review.

**Review by Oats87 — APPROVED** [review]
LGTM; tested this and was able to download the corresponding docker-machine drivers from `https://<server-url>/assets/docker-machine-driver-linode` and `https://<server-url>/assets/docker-machine-driver-harvester`

**Pull request merged**
